src/Artifacts.jl

```diff
@@ -71,12 +71,17 @@ function _mv_temp_artifact_dir(temp_dir::String, new_path::String)::Nothing
         # However, `mv` defaults to `cp` if `rename` returns an error.
         # `cp` is not atomic, so avoid the potential of calling it.
         err = ccall(:jl_fs_rename, Int32, (Cstring, Cstring), temp_dir, new_path)
-        # Ignore rename error, but ensure `new_path` exists.
-        if !isdir(new_path)
-            error("$(repr(new_path)) could not be made")
+        if err ≥ 0
+            # rename worked
+            chmod(new_path, filemode(dirname(new_path)))
+            set_readonly(new_path)
+        else
+            # Ignore rename error, if `new_path` exists.
+            if !isdir(new_path)
+                @error "your anti-virus may be interfering with artifact installation"
```

Comment:
I think this would be helpful, perhaps even point to the issue and/or put it inside an ^if windows` (I guess this is only a Windows issue)?

Reply:
Okay, I update the message to point to the github issue. I don't think there is a need to add an if windows because rename can fail on any OS in theory.